avro/src/de.rs

```diff
@@ -367,6 +369,7 @@ impl<'a, 'de> de::Deserializer<'de> for &'a Deserializer<'de> {
             }
             Value::Uuid(ref u) => visitor.visit_bytes(u.as_bytes()),
             Value::Decimal(ref d) => visitor.visit_bytes(&d.to_vec()?),
+            Value::BigDecimal(ref d) => visitor.visit_string(d.to_string()),
```

Comment:
This will never match because `deserialize_bytes` is not invoked for bigdecimal? It is either `deserialize_any` or `deserialize_str` (if feature `string-only` is enabled)?

Reply:
(So it has to be added to `deserialize_str` in order to get `string-only` to work as expected)

Reply:
Thanks for the review!
Feel free to open an alternative PR!